frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/FeatureOverviewEnvironmentMetrics/FeatureOverviewEnvironmentMetrics.tsx

```diff
@@ -0,0 +1,90 @@
+import type { IFeatureEnvironmentMetrics } from 'interfaces/featureToggle';
+import { calculatePercentage } from 'utils/calculatePercentage';
+import PercentageCircle from 'component/common/PercentageCircle/PercentageCircle';
+import { PrettifyLargeNumber } from 'component/common/PrettifyLargeNumber/PrettifyLargeNumber';
+import { styled } from '@mui/material';
+
+interface IFeatureOverviewEnvironmentMetrics {
+    environmentMetric?: IFeatureEnvironmentMetrics;
+    disabled?: boolean;
+}
+
+const StyledContainer = styled('div')({
+    marginLeft: 'auto',
+    display: 'flex',
+    alignItems: 'center',
+});
```

Comment:
I think this would be a perfect use case for the [`figure`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) component, using the info as the `figcaption`.